Sources/Testing/ExitTests/ExitTest.swift

```diff
@@ -114,7 +114,7 @@ extension ExitTest {
     // As with Linux, disable the generation core files. FreeBSD does not, as
     // far as I can tell, special-case RLIMIT_CORE=1.
     var rl = rlimit(rlim_cur: 0, rlim_max: 0)
-    _ = setrlimit(CInt(RLIMIT_CORE.rawValue), &rl)
+    _ = setrlimit(CInt(RLIMIT_CORE), &rl)
```

Comment:
If you remove the cast to `CInt`, does this line compile?

Reply:
it does, changed accordingly.